src/core/Form/Combobox/ComboboxItem/ComboboxItem.baseStyles.tsx

```diff
@@ -0,0 +1,70 @@
+import { css } from 'styled-components';
+import { defaultThemeTokens as theme } from '../../../theme';
+import { font } from '../../../theme/reset';
+
+export const baseStyles = css`
+  &.fi-combobox-item {
+    &:focus {
+      outline: none;
+    }
+
+    & .fi-checkbox {
+      padding: 10px;
+      user-select: none;
+      ${font(theme)('actionElementInnerText')}
+
+      & mark {
+        background-color: transparent;
+        font-weight: bold;
+      }
+
+      & .fi-checkbox_icon {
+        pointer-events: none;
+      }
+    }
+
+    & .fi-checkbox--checked {
+      background-color: ${theme.colors.depthSecondaryDark1};
+      color: ${theme.colors.blackBase};
+
+      & mark {
+        color: ${theme.colors.blackBase};
+      }
+    }
+
+    &--currentSelection {
+      & .fi-checkbox {
+        background-color: ${theme.colors.highlightBase};
+        color: ${theme.colors.whiteBase};
+
+        & mark {
+          color: ${theme.colors.whiteBase};
+        }
+
+        &.fi-checkbox--checked {
```

Comment:
Should this also be a descendant selector `& .fi-...`like other similar styles seem to be?

Reply:
Can not say straight away, need to check it.

What did you find when you tested to change it?

Reply:
I didn't test changing, it just caught my eye as it was different from other selectors.

Reply:
This is as it should be, --checked class is applied to the same element with fi-checkbox.